src/cursor-widget.ts

```diff
@@ -106,6 +193,11 @@ export class CursorWidget implements ICursorWidget {
 
     messageNode.style.borderColor = this._colorWithCSSVars("border");
     messageNode.style.backgroundColor = this._colorWithCSSVars("bg");
+    messageNode.style.color = Utils.getTextColor(this._color);
```

Comment:
What's your take on CSS Vars here?

Reply:
i can change it to the above format, which will give us customisation if needed and at the same time also takes care of setting a good enough color for text

Reply:
I meant, above usage as well. Do you think it's helpful or overkill?

Reply:
i dont think it would make sense for color specifically because the variable name is generic
`--color-${property}-${CursorWidget.WIDGET_NODE_CLASSNAME}`

it makes sense for properties like fontSize etc which would remain the same across widgets, compared to color which would change per user.

Reply:
In that case, get rid of them. We can revisit if needed later.